.clang-tidy

```diff
@@ -195,12 +204,23 @@ CheckOptions:
     value: ''
   - key: readability-identifier-naming.TypeTemplateParameterCase
     value: CamelCase
+  # workaround for https://github.com/llvm/llvm-project/issues/46097
+  #- key: readability-identifier-naming.TypeTemplateParameterIgnoredRegexp
+  #  value: expr-type
```

Comment:
Do we need this comment anymore if its combined with the lower entry?

Reply:
perhaps we should add a todo to remove when the upstream issue is solved